### PR TITLE
LRSD-7363 [LMS] Style inconsistencies in Learn

### PR DESCRIPTION
**Pull request by GabrielFernandoS**

### CODE REVIEW

**Comment by rtienpal** [comment]
ci:test:sf

**Comment by rtienpal** [comment]
ci:test:sf

File: workspaces/liferay-learn-workspace/client-extensions/liferay-learn-global-css/assets/global.css [hunk 1]
```diff
@@ -888,6 +901,12 @@ pre[class*='language-'] {
 	}
 }
 
+.learn-category div {
+	color: var(--color-neutral-10, #282934);
+  font-weight: 400;
```
Comment (joseabelenda) [inline workspaces/liferay-learn-workspace/client-extensions/liferay-learn-global-css/assets/global.css:906]: @GabrielFernandoS please align it.
SF

File: workspaces/liferay-learn-workspace/client-extensions/liferay-learn-global-css/assets/global.css [hunk 2]
```diff
@@ -1087,6 +1106,18 @@ pre[class*='language-'] {
 	}
 }
 
+.panel-title {
```
Comment (joseabelenda) [inline workspaces/liferay-learn-workspace/client-extensions/liferay-learn-global-css/assets/global.css:1109]: Sort

File: workspaces/liferay-learn-workspace/client-extensions/liferay-learn-global-css/assets/global.css [hunk 3]
```diff
@@ -1087,6 +1106,18 @@ pre[class*='language-'] {
...
+	color: var(--color-neutral-10, #282934) !important;
+	font-size: 1.125rem;
+	text-transform: none;
+}
+
+.treeview .btn {
```
Comment (joseabelenda) [inline workspaces/liferay-learn-workspace/client-extensions/liferay-learn-global-css/assets/global.css:1115]: Sort

File: workspaces/liferay-learn-workspace/client-extensions/liferay-learn-global-css/assets/global.css [hunk 4]
```diff
@@ -1440,7 +1471,14 @@ pre[class*='language-'] {
 				}
 			}
 		}
-	}
+	} 
+}
```
Comment (joseabelenda) [inline workspaces/liferay-learn-workspace/client-extensions/liferay-learn-global-css/assets/global.css:1475]: Remove space

File: workspaces/liferay-learn-workspace/client-extensions/liferay-learn-global-css/assets/global.css [hunk 5]
```diff
@@ -1440,7 +1471,14 @@ pre[class*='language-'] {
...
+
+.input-group .search-bar-keywords-input::placeholder {
```
Comment (joseabelenda) [inline workspaces/liferay-learn-workspace/client-extensions/liferay-learn-global-css/assets/global.css:1477]: Sort

**Review by joseabelenda — CHANGES_REQUESTED** [review]
@GabrielFernandoS please read my comments.
Thx

**Comment by lluk4z** [comment]
ci:test:sf

**Comment by joseabelenda** [comment]
resent:https://github.com/brianchandotcom/liferay-portal/pull/157072

**Pull request closed without merging**